Source/websocket/WebTransfer.h

```diff
@@ -300,17 +306,26 @@ namespace Web {
 
             return (result);
         }
-        inline uint64_t FileSize() const {
+
+        inline void LoadFileInfo()
```

Comment:
Where are you referring to this new function ? What is the purpose of this ?

Reply:
Since Loading is taking a long time, so just wanted to inform the caller a progress or indication about it

Reply:
Sorry, this was added testing purpose, will remove

Reply:
removed